packages/amplify-cli/src/commands/drift-detection/services/drift-formatter.ts

```diff
@@ -0,0 +1,705 @@
+        drifts: nestedDrifts,
+        template: nestedTemplate,
+      });
+    }
+  }
+
+  /**
+   * Format drift results based on the specified display format
+   */
+  public formatDrift(format: DriftDisplayFormat = 'tree'): {
+    summaryDashboard: string;
+    treeView?: string;
+    detailedChanges?: string;
+    categoryBreakdown?: string;
+    totalDrifted: number;
+  } {
+    const summaryDashboard = this.createSummaryDashboard();
+
+    let treeView: string | undefined;
+    let detailedChanges: string | undefined;
+    let categoryBreakdown: string | undefined;
+
+    switch (format) {
+      case 'tree':
+        treeView = this.createTreeView();
+        detailedChanges = this.createDetailedChanges();
+        categoryBreakdown = this.createCategoryBreakdown();
+        break;
+      case 'summary':
+        categoryBreakdown = this.createCategoryBreakdown();
+        break;
+      case 'json':
+        // JSON format handled separately in CLI
+        break;
+    }
+
+    return {
+      summaryDashboard,
+      treeView,
+      detailedChanges,
+      categoryBreakdown,
+      totalDrifted: this.summary.totalDrifted,
+    };
+  }
+
+  /**
+   * Create simplified JSON output
+   */
+  public createSimplifiedJsonOutput(): SimplifiedJsonOutput {
+    return {
+      stackName: this.rootStackName,
+      numResourcesWithDrift: this.summary.totalDrifted,
+      numResourcesUnchecked: this.summary.totalUnchecked,
+      timestamp: new Date().toISOString(),
+    };
+  }
+
+  /**
+   * Create a summary dashboard with overall statistics
+   */
+  private createSummaryDashboard(): string {
+    const projectName = this.extractProjectName(this.rootStackName);
+
+    const border = '─'.repeat(DISPLAY_CONSTANTS.BORDER_WIDTH);
+    let dashboard = '';
+
+    dashboard += chalk.cyan(`┌${border}┐\n`);
+    dashboard += chalk.cyan(
+      `│${' '.repeat(DISPLAY_CONSTANTS.TITLE_PADDING)}DRIFT DETECTION SUMMARY${' '.repeat(DISPLAY_CONSTANTS.TITLE_PADDING)}│\n`,
+    );
+    dashboard += chalk.cyan(`├${border}┤\n`);
+
+    // Project name
+    dashboard += this.formatDashboardLine('Project', chalk.bold(projectName), DISPLAY_CONSTANTS.LABEL_PROJECT);
+
+    // Total stacks
+    dashboard += this.formatDashboardLine(
+      'Total Stacks Checked',
+      chalk.bold(this.summary.totalStacks),
+      DISPLAY_CONSTANTS.LABEL_STACKS_CHECKED,
+    );
+
+    // Resources with drift
+    const driftedColor = this.summary.totalDrifted > 0 ? chalk.red : chalk.green;
+    dashboard += this.formatDashboardLine(
+      'Resources with Drift',
+      driftedColor(this.summary.totalDrifted),
+      DISPLAY_CONSTANTS.LABEL_RESOURCES_DRIFT,
+    );
+
+    // Resources in sync
+    dashboard += this.formatDashboardLine(
+      'Resources in Sync',
+      chalk.green(this.summary.totalInSync),
+      DISPLAY_CONSTANTS.LABEL_RESOURCES_SYNC,
+    );
+
+    // Unchecked resources
+    dashboard += this.formatDashboardLine(
+      'Unchecked Resources',
```

Comment:
Lets print the unchecked resources so the user can see they are less critical (it would be bad if we cant check a stateful resource).